adapters/pubmatic/pubmatictest/supplemental/impExtData.json

```diff
@@ -0,0 +1,156 @@
+{
+    "mockBidRequest": {
+        "id": "test-request-id",
+        "imp": [
+            {
+                "id": "test-imp-id",
+                "banner": {
+                    "format": [
+                        {
+                            "w": 300,
+                            "h": 250
+                        }
+                    ]
+                },
+                "ext": {
+                    "bidder": {
+                        "adSlot": "AdTag_Div1@300x250",
+                        "publisherId": "999",
+                        "wrapper": {
+                            "version": 1,
+                            "profile": 5123
+                        },
+                        "dctr": "k1=v1|k2=v2"
+                    },
+                    "data": {
+                        "adserver": {
+                            "name": "gam",
+                            "adslot": "/1111/home"
+                        },
+                        "pbadslot": "/2222/home",
+                        "sport": [
+                            "rugby",
+                            "cricket"
+                        ]
+                    }
+                }
+            }
+        ],
+        "device": {
+            "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36"
+        },
+        "site": {
+            "id": "siteID",
+            "publisher": {
+                "id": "1234"
+            }
+        }
+    },
+    "httpCalls": [
+        {
+            "expectedRequest": {
+                "uri": "https://hbopenbid.pubmatic.com/translator?source=prebid-server",
```

Comment:
Nitpick: Can we use a fake url here? Something that at first sight is easily understood to be a URL for testing purposes.

Reply:
This is the expected SSP URL. I can see that rest of the adapters also use original URL. 
Do I need to change https://github.com/prebid/prebid-server/blob/2eb2deec2e1600c070d049c5333f94fa8089c52b/adapters/pubmatic/pubmatic_test.go#L20 for this, this will require changes in all the existing JSON files?

Reply:
This is the expected SSP URL. I can see that rest of the adapters also use original URL. 
Do I need to change https://github.com/prebid/prebid-server/blob/2eb2deec2e1600c070d049c5333f94fa8089c52b/adapters/pubmatic/pubmatic_test.go#L20 for this, this will require changes in all the existing JSON files?